packages/mds-geography-author/tests/api.spec.ts

```diff
@@ -499,45 +458,45 @@ describe('Tests app', () => {
         })
     })
 
-    it('verifies cannot PUT geography with a publish_date', done => {
-      const geography = {
-        name: 'foo',
-        geography_id: GEOGRAPHY_UUID,
-        publish_date: 1589817834000,
-        geography_json: LA_CITY_BOUNDARY
-      }
-      request
-        .put(pathPrefix(`/geographies/${geography.geography_id}`))
-        .set('Authorization', GEOGRAPHIES_WRITE_SCOPE)
-        .send(geography)
-        .expect(400)
-        .end((err, result) => {
-          test.assert(result.body.error.name === `ValidationError`)
-          test.assert(result.body.error.info[0].params.additionalProperty.includes('publish_date'))
-          test.value(result).hasHeader('content-type', APP_JSON)
-          done(err)
-        })
-    })
-
-    it('verifies cannot POST geography with a publish_date', done => {
-      const geography = {
-        name: 'foo',
-        geography_id: GEOGRAPHY_UUID,
-        publish_date: 1589817834000,
-        geography_json: LA_CITY_BOUNDARY
-      }
-      request
-        .post(pathPrefix(`/geographies`))
-        .set('Authorization', GEOGRAPHIES_WRITE_SCOPE)
-        .send(geography)
-        .expect(400)
-        .end((err, result) => {
-          test.assert(result.body.error.name === `ValidationError`)
-          test.assert(result.body.error.info[0].params.additionalProperty.includes('publish_date'))
-          test.value(result).hasHeader('content-type', APP_JSON)
-          done(err)
-        })
-    })
+    // it('verifies cannot PUT geography with a publish_date', done => {
```

Comment:
Can these tests be fixed? Should they be removed?

Reply:
that comment should have had a `//FIXME` on it.

Reply:
updated!